index.js

```diff
@@ -97,6 +101,19 @@ module.exports = {
     'computed-property-spacing': [2, 'never'],
     'key-spacing': [2, {beforeColon: false, afterColon: true, mode: 'minimum'}],
     'brace-style': [2, '1tbs', {allowSingleLine: true}]
+    'switch-colon-spacing': [2, {after: true, before: false}],
+    'array-bracket-newline': [2, {multiline: true}],
+    'array-element-newline': [2, {multiline: true}],
+    'capitalized-comments': [2, 'always', {ignoreConsecutiveComments: true}],
```

Comment:
I just check some examples of my comments. Why we put so many restrictions on them?
I think I never started my comment with capital - especially single line comments